fairseq/data/audio/raw_audio_dataset.py

```diff
@@ -11,11 +11,17 @@
 import numpy as np
 import torch
 import torch.nn.functional as F
+try:
+    import soundfile as sf
+except ImportError as err:
+    print(err)
+    print('try: pip install soundfile')
```

Comment:
this is going to get printed out for other people who dont use speech related functionality, since we read all datasets in data/__init_\_

Reply:
fixed